crds/roman/proto-cache/mappings/roman/roman_wfi_flat_0001.rmap

```diff
@@ -0,0 +1,21 @@
+header = {
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'roman_wfi_flat_0000.rmap',
+    'description' : 'Hand edited for roman baseline 2020-10-09.   Prototype / fake sample only,  update for Roman.',
+    'filekind' : 'FLAT',
+    'filetype' : 'FLAT',
+    'instrument' : 'WFI',
+    'mapping' : 'REFERENCE',
+    'name' : 'roman_wfi_flat_0000.rmap',
+    'observatory' : 'ROMAN',
+    'parkey' : (('META.INSTRUMENT.DETECTOR', 'META.INSTRUMENT.FILTER', 'META.INSTRUMENT.PUPIL'), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
```

Comment:
Someone needs to write this down somewhere other than CRDS code so it can be reviewed and agreed upon by the people who know.

Reply:
Yes, this will be part of a document that lives with the romancal documentation. At least this is the current plan.
However, it may be some time before this document is created. This shouldn't stop work on crds.